docs/ignoring-files-folders-code.md

```diff
@@ -66,47 +64,54 @@ Semgrep provides several methods to customize ignore behavior. Refer to the foll
 
 | Goal | Method |
 |:---- |:------ |
-| To scan all files within Semgrep's scope each time you run Semgrep (only files within `.git` will be ignored). | Create an empty `.semgrepignore` file in your repository root directory or in your project's working directory. An empty `.semgrepignore` will make Semgrep scan paths in `.gitignore`. |
+| To scan all files within Semgrep's scope each time you run Semgrep (only files within `.git` are ignored). | Create an empty `.semgrepignore` file in your repository root directory or in your project's working directory. If your `.semgrepignore` file is empty, Semgrep scans only paths in `.gitignore`. |
```

Comment:
Is this change here correct? That if the `.semgrepignore` is blank, semgrep will only scan the paths in the `.gitignore`? That doesn't seem right.

Reply:
Thanks for a good catch @sabrinabrogren! The original is not right as well: `An empty `.semgrepignore` will make Semgrep scan paths in `.gitignore`.` Maybe it means, that when you do not include an empty `.semgreignore` file, Semgrep will use `.gitignore` instead to see what it has to ignore?

Reply:
That would make sense to me. Let me post this in the #team-app channel, I'm not sure about this.

Reply:
```suggestion
| To scan all files within Semgrep's scope each time you run Semgrep (only files within `.git` are ignored). | Create an empty `.semgrepignore` file in your repository root directory or in your project's working directory. An empty `.semgrepignore` will make Semgrep scan paths in `.gitignore`. |
```